src/main/java/net/sf/jabref/importer/ImportFileFilter.java

```diff
@@ -42,11 +51,17 @@ public ImportFormat getImportFormat() {
 
     @Override
     public boolean accept(File file) {
-        return true;
-        /*if (file.isDirectory())
+        if (format.getExtensions() == null || format.getExtensions().isEmpty()) {
```

Comment:
I know, but your colleagues implemented `getExtensions` for all importers and activated the corresponding test in #1487. Hence we will merge #1487 first and then this PR (without the null checks)